### PR TITLE
chore: generic minor fixes

### PR DESCRIPTION
**Pull request by mlavacca**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
The `--input_file` flag has been renamed `--input-file`.
```

### CODE REVIEW

File: cmd/print.go [hunk 1]
```diff
@@ -228,25 +229,25 @@ func newPrintCommand() *cobra.Command {
...
+		RunE:  pr.PrintGatewayAPIObjects,
 	}
 
 	cmd.Flags().StringVarP(&pr.outputFormat, "output", "o", "yaml",
-		fmt.Sprintf(`Output format. One of: (%s)`, strings.Join(allowedFormats, ", ")))
+		fmt.Sprintf(`Output format. One of: (%s).`, strings.Join(allowedFormats, ", ")))
 
-	cmd.Flags().StringVar(&pr.inputFile, "input_file", "",
-		`Path to the manifest file. When set, the tool will read ingresses from the file instead of reading from the cluster. Supported files are yaml and json`)
+	cmd.Flags().StringVar(&pr.inputFile, "input-file", "",
```
Comment (levikobi) [inline cmd/print.go:239]: It's about time this one is changed
Please update the README with this change as well, as it currently mentions we have a `--input_file` flag

Comment (mlavacca) [reply cmd/print.go:239]: yep, good catch. Updated in [d886474](https://github.com/kubernetes-sigs/ingress2gateway/pull/156/commits/d886474abeaf455aec00f9b818f979b6b422f99f)

**Pull request merged**